crates/nu-cli/Cargo.toml

```diff
@@ -24,11 +24,11 @@ nu-utils = { path = "../nu-utils", version = "0.78.1" }
 nu-color-config = { path = "../nu-color-config", version = "0.78.1" }
 
 nu-ansi-term = "0.47.0"
-reedline = { version = "0.18.0", features = ["bashisms", "sqlite"] }
```

Comment:
Same

Reply:
Hi, I get a warning message:
warning: patch for the non root package will be ignored, specify patch at the workspace root

It seems that we don't need to change `nu-cli/Cargo.toml`.